Classes/ios/Scanners/MTBBarcodeScanner.m

```diff
@@ -455,12 +467,6 @@ - (AVCaptureSession *)newSessionWithCaptureDevice:(AVCaptureDevice *)captureDevi
     return newSession;
 }
 
-- (AVCaptureDeviceInput *)deviceInputForCaptureDevice:(AVCaptureDevice *)captureDevice error:(NSError **)error {
```

Comment:
This method didn't do anything but call another method with the exact same parameters